package.json

```diff
@@ -8,22 +8,28 @@
     "eject": "expo eject"
   },
   "dependencies": {
+    "@mapbox/polyline": "^1.1.0",
```

Comment:
do we need all those dependencies? just for single map?

Reply:
polyline is actually required to be able to draw the line between two points in the map. But I will definitely check if there are some dependencies that can be removed !